code/turf/floors.dm

```diff
@@ -861,12 +861,45 @@ DEFINE_FLOORS(marble/border_wb,
 	mat_appearances_to_ignore = list("steel","synthrubber","glass")
 	step_material = "step_wood"
 	step_priority = STEP_PRIORITY_MED
-	mat_changename = 0
+	mat_changename = FALSE
 
 	New()
+		var/image/I
+		#ifdef UNDERWATER_MAP
+		I = image('icons/turf/outdoors.dmi', "sand_other")
```

Comment:
The sand has random icons too, why not here like space below?

Reply:
Sure, might've just misread code originally, idk